src/core/lib/iomgr/timer.c

```diff
@@ -278,7 +278,8 @@ static int refill_queue(shard_type *shard, gpr_timespec now) {
   return !grpc_timer_heap_is_empty(&shard->heap);
 }
 
-/* This pops the next non-cancelled timer with deadline <= now from the queue,
+/* This pollent the next non-cancelled timer with deadline <= now from the
```

Comment:
Ugh, yes. The perils of global sed search-and-replace. I'll send a fixing PR momentarily.